Cargo.toml

```diff
@@ -18,7 +18,7 @@ default = [ "std", "secp-recovery" ]
 base64 = [ "base64-compat" ]
 unstable = []
 rand = ["secp256k1/rand-std"]
-use-serde = ["serde", "bitcoin_hashes/serde", "secp256k1/serde"]
+serde = ["actual_serde", "bitcoin_hashes/serde", "secp256k1/serde"]
```

Comment:
nit: Perhaps use `-` instead of `_` to be consistent with other names?

Reply:
+1 from me (I often wish we had `bitcoin-hashes` too but its probably too much churn to replace `bitcoin_hashes`).

Reply:
My naming is usually `_` where I would've put space and `-` where I would've put `-` in normal text. Although it sometimes isn't a clear cut - e.g. `bitcoin_hashes` :D

Anyway, will rename, I also forgot to document people must not rely on the name.

Reply:
I like `-` because it reminds me of writing Lisp :) 

Looks like you changed it in the manifest but forgot to change all the `actual_serde` instances in  the derives. That wasn't intentional, right?

Reply:
Yes, it has to be `_` in the code since `extern crate actual-serde;` wouldn't compile.